tests/specs/lemmas.k

```diff
@@ -314,6 +341,16 @@ module LEMMAS-HASKELL [symbolic, kore]
     // (pop)
     rule X <=Int chop ( X +Int Y ) => false  requires pow256 <=Int ( X +Int Y ) andBool #rangeUInt(256, X) andBool #rangeUInt(256, Y) [simplification]
 
+  // ########################
+  // Keccak
+  // ########################
+
+    rule { I:Int #Equals keccak(_) } => #Bottom requires 0 <=Int I andBool I <Int 10 [simplification]
+
+    rule { I1:Int #Equals keccak(_) +Int I2 } => #Bottom 
+      requires 0 <=Int I1 andBool I1 <Int 10 andBool 0 <=Int I2 andBool I2 <Int 10 
+      [simplification]
+
```

Comment:
Once again, these can probably be made into explicit dis-equalities on the pre-condition of the specific proofs it affects.

Reply:
Same as above...

Reply:
It was agreed to move these simplifications as-is to the `mcd` folder, and out of the general lemmas.